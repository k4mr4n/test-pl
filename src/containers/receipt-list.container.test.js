import React from 'react'
import { createStore } from 'redux'

import ReceiptListContainer from './receipt-list.container'
import { renderWithRedux } from '../utils/testing.util'

describe('Receipt List', () => {
  test('should render receipt list', () => {
    const receiptList = [{ description: 'dsc', amount: 100, currency: 'CAD' }]
    const store = createStore(() => ({ receiptList: { list: receiptList } }))
    const { baseElement } = renderWithRedux(<ReceiptListContainer />, {
      store
    })
    expect(baseElement).toMatchSnapshot()
  })
})
