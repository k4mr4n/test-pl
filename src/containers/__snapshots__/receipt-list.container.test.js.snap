// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Receipt List should render receipt list 1`] = `
<body>
  <div>
    <div>
      <h1>
        Receipts:
      </h1>
      <ul>
        <li>
          dsc
           - 
          100
           - 
          CAD
        </li>
      </ul>
    </div>
  </div>
</body>
`;
